File: MedLabO.Models/Administrator.cs
namespace MedLabO.Models
{
    public class Administrator
    {
        public string Id { get; set; }
        public string? Ime { get; set; }
        public string? Prezime { get; set; }
        public bool IsKontakt { get; set; } = false;
        public string? KontaktInfo { get; set; }
    }
}

File: MedLabO UI/medlabo_mobile/lib/models/administrator/administrator.dart
import 'package:json_annotation/json_annotation.dart';

part 'administrator.g.dart';

@JsonSerializable()
class Administrator {
  String? id;
  String? ime;
  String? prezime;
  bool? isKontakt;
  String? kontaktInfo;
  String? userName;
  String? email;
  String? phoneNumber;

  Administrator(
      {this.id,
      this.ime,
      this.prezime,
      this.isKontakt,
      this.kontaktInfo,
      this.userName,
      this.email,
      this.phoneNumber});

  factory Administrator.fromJson(Map<String, dynamic> json) =>
      _$AdministratorFromJson(json);

  Map<String, dynamic> toJson() => _$AdministratorToJson(this);
}
